ts/screens/authentication/LandingScreen.tsx

```diff
@@ -296,133 +319,232 @@ export const LandingScreen = () => {
     }
   }, [isCieSupported, navigation]);
 
-  const renderCardComponents = () => {
-    const cardProps = getCards(isCieSupported());
-    return cardProps.map(p => (
-      <LandingCardComponent key={`card-${p.id}`} {...p} />
-    ));
+  const renderCardComponents = React.useCallback(
+    () =>
+      carouselCards.map(p => (
+        <LandingCardComponent
+          ref={p.id === 5 ? accessibilityFirstFocuseViewRef : null}
+          screenDimensions={screenDimension}
+          key={`card-${p.id}`}
+          {...p}
+        />
+      )),
+    [screenDimension]
+  );
+
+  const handleContinueWithRootOrJailbreak = React.useCallback(
+    (continueWith: boolean) => {
+      dispatch(continueWithRootOrJailbreak(continueWith));
+    },
+    [dispatch]
+  );
+
+  const CarouselDots = () => {
+    const dotTouchCount = React.useRef(0);
+
+    const blueColor =
+      colorScheme === "dark"
+        ? IOColors.white
+        : isDesignSystemEnabled
+        ? IOColors["blueIO-500"]
+        : IOColors.blue;
+
+    return (
+      <View
+        importantForAccessibility="yes"
+        accessibilityElementsHidden={false}
+        style={styles.indicatorContainer}
+        onTouchEnd={(_: GestureResponderEvent) => {
+          // eslint-disable-next-line functional/immutable-data
+          dotTouchCount.current++;
+          if (dotTouchCount.current === 3) {
+            // eslint-disable-next-line functional/immutable-data
+            dotTouchCount.current = 0;
+            navigateToCieUatSelectionScreen();
+          }
+        }}
+      >
+        {carouselCards.map((_, imageIndex) => {
+          const width = scrollX.interpolate({
+            inputRange: [
+              windowWidth * (imageIndex - 1),
+              windowWidth * imageIndex,
+              windowWidth * (imageIndex + 1)
+            ],
+            outputRange: [8, 16, 8],
+            extrapolate: "clamp"
+          });
+          const backgroundColor = scrollX.interpolate({
+            inputRange: [
+              windowWidth * (imageIndex - 1),
+              windowWidth * imageIndex,
+              windowWidth * (imageIndex + 1)
+            ],
+            outputRange: [
+              IOColors["grey-200"],
+              blueColor,
+              IOColors["grey-200"]
+            ],
+            extrapolate: "clamp"
+          });
+          return (
+            <Animated.View
+              key={imageIndex}
+              style={[styles.normalDot, { width, backgroundColor }]}
+            />
+          );
+        })}
+      </View>
+    );
   };
 
-  const handleContinueWithRootOrJailbreak = (continueWith: boolean) => {
-    dispatch(continueWithRootOrJailbreak(continueWith));
+  const Carousel = () => {
```

Comment:
I would propose that you move the Carousel component to another file.
I propose this modification because
1. this file has about 600 lines and is difficult to manage and understand
2. the carousel is one of the possible UIs that can display this screen 
3. LandingCardComponent was created for the same logic

Reply:
Why not! I'll try it out!

Reply:
This was a gr8 suggestion! It's much better now: 21a54e8b3328eb778957cc5000ff29e7e7ca432e , what do you think?

During this refactoring I realize that the tracking of the first carousel page is not necessary, and I removed it. If you look at the `trackCarousel` function it doesn't take care of it.

I took the chance to remove the function that tracked the removed carousel page.

In this commit I also address the [new buttons spacing](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3?node-id=4358:56120&mode=design#714623704) and the removal of the old graphics as @dmnplb suggested.